docs/dasharo-menu-docs/overview.md

```diff
@@ -10,10 +10,125 @@ subsections you will get to know:
+    - [One Time Boot](#one-time-boot)
+    - [Boot Maintenance Manager](boot-maintenance-mgr.md)
+
+* [Boot Manager Menu](#boot-manager-menu) - entered with a different key than
+  used for setup application. Lists all bootable options and allows one to
+  override the boot path.
+
+### Main Page
+
+![](/images/menus/main_page.jpeg){ class="center" }
+
+The page is the main view of the firmware setup application. It contains the
+board model (`MS-7D25`), installed CPU and firmware version in the top-left
+corner. In the top-right corner the CPU frequency and system RAM amount are
+shown.
+
+From the main page one may access all menus and submenus available in the
+firmware setup. Besides the menus there is also an option to:
+
+1. Change the language (currently only English is supported)
+2. `Continue` - execute the top first boot order priority
+3. `Reset` - resets the platform.
+
+The currently available menus/submenus are as follows:
+
+* [User Password Management](#user-password-management) - allows to set
+  firmware setup password
+* [Device Manager](device-manager.md) - allows configuring various devices and
+  features like: UEFI Secure Boot, TPM device, SATA and TCG OPAL password, etc.
+  It may also contain informational menus like Driver Health Manager, Network
+  Device List and others.
+* [Dasharo System Features](dasharo-system-features.md)- contains submenus for
+  features specific to Dasharo products and Dasharo supported platforms
+* [One Time Boot](#one-time-boot) - allows to choose which boot entry to
+  execute. It simply lists all available boot options and allows to select one
+  the same way as [Boot Manager Menu](#boot-manager-menu)
+* [Boot Maintenance Manager](boot-maintenance-mgr.md) - allows to manipulate
+  various UEFI standard variables responsible for console and boot options. One
+  may choose which devices should be used for input and output, choose to boot
+  an arbitrary file or modify the boot options and boot order.
+
+NOTE: not all submenus may be available on your platform.
+[Contact Dasharo Team](mailto:contact@dasharo.com) for more information and
+possible feature extension of your platform.
+
+### User Password Management
+
+User Password Management menu allows one to set firmware setup admin password:
+
+![](/images/menus/password_mgr.jpeg){ class=center" }
+
+The password must be a strong one. The requirements are described on the help
+section on the right side of the menu page. If the password is too weak it will
+not be accepted. The page shows the current status of `Admin Password` if it
+was provisioned or not.
+
+To disable the password, enter the `User Password Management` menu again and
+request to set new password. You will be asked to enter current password. When
+a window to provide new password pops up, leave the field empty and confirm it
+twice with `ENTER`. The password installation status on the menu page should
+change to `Not Installed`.
```

Comment:
It would be good to add information, that if anyone would like to set a new password, it must be different from the last 5 passwords.

Reply:
@sulewskiprzemyslaw is this a recommendation or just how the feature works?

Reply:
@miczyg1 feature works in that way

Reply:
Thanks, added that information